maud/src/lib.rs

```diff
@@ -76,34 +76,44 @@ pub trait Render {
     }
 }
 
+#[cfg(feature = "unstable")]
 impl<T: fmt::Display + ?Sized> Render for T {
     default fn render_to(&self, w: &mut String) {
         let _ = write!(Escaper::new(w), "{}", self);
     }
 }
 
+impl<T: AsRef<str>> Render for PreEscaped<T> {
+    fn render_to(&self, w: &mut String) {
+        w.push_str(self.0.as_ref());
+    }
+}
+
+#[cfg(feature = "unstable")]
 impl Render for String {
     fn render_to(&self, w: &mut String) {
         let _ = Escaper::new(w).write_str(self);
     }
 }
 
+#[cfg(feature = "unstable")]
 impl Render for str {
     fn render_to(&self, w: &mut String) {
         let _ = Escaper::new(w).write_str(self);
     }
 }
 
-/// A wrapper that renders the inner value without escaping.
-#[derive(Debug, Clone, Copy)]
-pub struct PreEscaped<T: AsRef<str>>(pub T);
-
-impl<T: AsRef<str>> Render for PreEscaped<T> {
+#[cfg(not(feature = "unstable"))]
+impl<T: fmt::Display + ?Sized> Render for T {
     fn render_to(&self, w: &mut String) {
-        w.push_str(self.0.as_ref());
+        let _ = write!(Escaper::new(w), "{}", self);
     }
 }
```

Comment:
I meant moving this up to the top, not the `PreEscaped` impl... but no worries, I'll just merge this and fix it myself 😛